extensions/norwegian-bokmal/src/extension.ts

```diff
@@ -0,0 +1,57 @@
+'use strict';
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+import * as vscode from 'vscode';
+interface CodeSpellCheckerExtension {
+    registerConfig(path: string): Promise<void>;
+    enableLocal(isGlobal: boolean, locale: string): Promise<void>;
+    disableLocal(isGlobal: boolean, locale: string): Promise<void>;
+}
+
+
+const locale = 'nb';
```

Comment:
After seeing this, I realize we got the name of the dictionary wrong. I'll change it.
The dictionary should be Language-Country, so no_NB should be nb_NO.

Reply:
I have fixed the dictionary on NPM.

Please change any references like `no_NB` or `no-nb` to `nb_NO` ans `nb-no` respectively.

Please add a `cspell.json` file in the extension directory with `"words": ["Bokmål"]` to fix the spelling complaints.

Please note: the dependency in `package.json` needs to be  `@cspell/dict-nb-no`

Reply:
Or add them to the `cusotm-words.txt` like you did.

Reply:
Done and done. The pipeline jobs should pass now